code/datums/mood_events/drug_events.dm

```diff
@@ -78,3 +78,8 @@
 	description = "<span class='nicegreen'>Eh ah AAAAH! HA HA HA HA HAA! Uuuh.</span>\n"
 	mood_change = 6
 	timeout = 3 MINUTES
+
+/datum/mood_event/antidepressant
+	description = "<span class='nicegreen'>All the bad things feel a bit more far away.</span>\n"

```

Comment:
is this newline character necessary what does it do

Reply:
can we just account for the newlines in wherever this is being consumed why do we need newlines on the end of descriptions thats stinky code @Qustinnus did you do this

Reply:
yea when mood was originally made i was still pretty new tbf